src/layer-composer/generators/track/track.ts

```diff
@@ -143,28 +140,35 @@ class TrackGenerator {
       id: config.id,
       source: config.id,
       layout: {},
-      paint: { 'line-color': config.color || 'rgba(0, 193, 231, .7)' },
+      paint: {
+        'line-color': config.color || this.defaultColor,
+        'line-opacity': 0.8,
+      },
       metadata: {
         group: Group.Track,
       },
     }
     const layers = [layer]
 
-    if (config.highlightedTime) {
-      const id = `${config.id}${this.highlightSufix}`
-      const highlightedLayer = getHighlightedLayer(id)
-      layers.push(highlightedLayer)
-    }
     if (config.highlightedEvent) {
       const id = `${config.id}${this.highlightEventSufix}`
       const paint = {
-        'line-color': config.highlightedEvent.color || 'rgba(0, 193, 231, 1)',
-        'line-width': config.highlightedEvent.width || 5,
+        'line-color': config.color || this.defaultColor,
```

Comment:
When is `this.defaultColor` accessed?
I'd be careful with using `this` on generators as it's easy to forget that it will apply to all instances of track

Reply:
> When is this.defaultColor accessed?

at the class definition: https://github.com/GlobalFishingWatch/layer-composer/pull/21/files#diff-a5967b6c99920c37e45156733abbf730R80

> I'd be careful with using this on generators as it's easy to forget that it will apply to all instances of track

I completely agree with the use of `this`, but in this case I proposed to use it because is a fallback in case the generator doesn't have a color and it could help to debug... otherwise we could set the color as a required param and force to send one, what do you prefer ?

Reply:
Why not a constant ?

Reply:
Ok, let's use a constant

Reply:
Thanks